### PR TITLE
[doc] Tutorials > User guide, design section [WIP]

### PR DESCRIPTION
**Pull request by amoutenet**
We change the Tutorials section in a "User guide" section that aims at giving a broad overview of the main classes and applications of TRIQS through code snippets ready-to-use.

The spirit of this section is explained in the additional paragraph of index.rst, where we tell new users to: 
- First install triqs
- Then do the Python tutorials available on the GitHub
- Browse through the User guide codes to get a sense of what is in the library

Of course the User guide section contains a warning saying that this is not meant to replace the full reference.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -53,3 +53,22 @@ interface with other programs can be done at the very user-friendly python
...
 
 
+Where to start?
+---------------
+
+First, visit our :ref:`Install <install>` section to obtain the TRIQS library.
+
+We then strongly recommend first-time users to go through the extensive set of Jupyter Notebook tutorials provided on `GitHub <https://github.com/TRIQS/tutorials>`_.
+You can run the notebooks either in an interactive `Binder Session <https://triqs.github.io/notebook>`_ on the web
+or run them locally after setting up triqs on your computer.
```
Comment (Wentzell) [inline doc/index.rst]:
```suggestion
or run them locally after setting up TRIQS on your computer.
```

File: doc/userguide.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Welcome to the TRIQS project!
+
+This guide aims at presenting you the main classes of TRIQS and its applications through ready-to-use examples. These small demo codes
+should help you understand the logic behind the general design and organization of the library.
+
+.. note::
+
+     We strongly recommend first-time users to go through the extensive set of Jupyter Notebook tutorials provided on `GitHub <https://github.com/TRIQS/tutorials>`_.
+     You can run the notebooks either in an interactive `Binder Session <https://triqs.github.io/notebook>`_ on the web
+     or run them locally after setting up triqs on your computer.
```
Comment (Wentzell) [inline doc/userguide.rst]:
```suggestion
     or run them locally after setting up TRIQS on your computer.
```

**Review by Wentzell — CHANGES_REQUESTED** [review]
Thank you Alice! I like the changes!
I think its great to have the getting started section already on the main-page.
I just had a few minor corrections to your commit.

**Comment by amoutenet** [comment]
@Wentzell thank you very much, here is the updated PR!

**Pull request merged**